dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
+          DependsOnTargets="FilterProjects">
+        <MSBuild Targets="ProducesPackageId"
+             Projects="@(Project)"
+             BuildInParallel="true"
+             ContinueOnError="ErrorAndContinue">
+          <Output TaskParameter="TargetOutputs"
+                  ItemName="PackageIds" />             
+        </MSBuild>
+    <ItemGroup>
+      <PackageIds Include="@(PackageIds)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="ProducesTarget" DependsOnTargets="FilterProjects">
```

Comment:
Oh, I understand what you are saying. I look at it as a constrain and not a bug but that doesn't mean we can't take a look at it and change it. It is our code so we can adapt it :)
I'll ile an issue for it.
Thanks Chris